.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+  "directory": ".."
```

Comment:
We no longer check in this Bower configuration, as it is very dangerous for unwary element developers. Please remove the checked-in `.bowerrc`.